src/pages/images/actions/ExportImageTarballBtn.tsx

```diff
@@ -0,0 +1,48 @@
+import { FC, useState } from "react";
+import { exportImage } from "api/images";
+import { LxdImage } from "types/image";
+import { ActionButton, Icon } from "@canonical/react-components";
+import { useToastNotification } from "context/toastNotificationProvider";
+import { handleDownloadByType } from "util/imageExport";
+
+interface Props {
+  image: LxdImage;
+}
+
+const ExportImageTarballBtn: FC<Props> = ({ image }) => {
+  const toastNotify = useToastNotification();
+  const [isLoading, setLoading] = useState(false);
+  const description = image.properties?.description ?? image.fingerprint;
+
+  const handleExport = async () => {
+    setLoading(true);
+    try {
+      await exportImage(image.fingerprint, (resp) =>
+        handleDownloadByType(resp),
+      );
+
+      toastNotify.success(
+        `Image ${description} was successfully downloaded. Please check your downloads folder.`,
+      );
+    } catch (e) {
+      toastNotify.failure(`Image ${description} was unable to download.`, e);
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  return (
+    <ActionButton
+      aria-label="export image"
```

Comment:
```suggestion
      title="export image"
```
This should be a visible title instead. The icon does not transport what this is doing well enough.

Reply:
In this case I would use both. `aria-label` is the preferred way for accessibility support.